types/openlayers/index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Type definitions for OpenLayers v4.3.0
+// Type definitions for OpenLayers v4.3.4
 // Project: http://openlayers.org/
 // Definitions by: Olivier Sechet <https://github.com/osechet>
 //                 Bin Wang <https://github.com/wb14123>
 //                 Junyoung Clare Jang <https://github.com/ailrun>
+//                 Alexandre Melard <https://github.com/mylen>
```

Comment:
If you add yourself as an author, you'll be notify of all the future updates in the file. It's fine for me but it's better for you to know.